exporters/otlp/common/src/test/java/io/opentelemetry/exporter/internal/otlp/logs/LogsRequestMarshalerTest.java

```diff
@@ -131,7 +128,6 @@ void toProtoLogRecord(MarshalerSource marshalerSource) {
     assertThat(logRecord.getTraceId().toByteArray()).isEqualTo(TRACE_ID_BYTES);
     assertThat(logRecord.getSpanId().toByteArray()).isEqualTo(SPAN_ID_BYTES);
     assertThat(logRecord.getSeverityText()).isEqualTo("INFO");
-    assertThat(logRecord.getEventName()).isEqualTo(EVENT_NAME);
```

Comment:
Why remove these event name assertions?

Reply:
We split out the tests for experimental features which rely on the incubator vs. stable features which do not. The main test suite intentionally omits a dependency on `opentelemetry-api-incubator` to ensure that everything works even without the artifact on the classpath. The `testIncubating` suite includes `opentelemetry-api-incubator` and verifies the experimental features. 

This assertion now lives over here: https://github.com/open-telemetry/opentelemetry-java/pull/7123/files#diff-e40b52ea213ceff0507b8f5ae9ebd5fa6d4cbfa5e555c5bd86e993cb7e164dc2R81-R149

`event.name` assertions were sneaking into the main test suite but this was an oversight.